### ISSUE TITLE
Add Setting (Sub)Category for Accessibility Options

### ISSUE THREAD

**Issue by FlameSoulis** [Yellow-Dog-Man/Resonite-Issues#3040]
### Is your feature request related to a problem? Please describe.

Currently, all accessibility options are hidden away with the bot commands, which are still case-sensitive, not easy to access, and don't provide much feedback on errors. As such, important ones, such as the Potato and sensory badges, are effectively 'locked off' unless you know the magic words and ritual, and given how valuable the Potato badge could be (especially if you are checking for it to adjust world quality similar to #2327), this is pretty troubling when I have to explain DOS/UNIX commands to new people and friends.

### Describe the solution you'd like

One idea would be to add a category for Accessibility options or a subcategory to existing ones (Profile/Misc?). If the bot is strictly the only way to adjust the parameters, a cheesy workaround would be to execute the chat commands on the user's behalf or just allow the account edits as needed.

### Describe alternatives you've considered

There are no alternatives that resolve easier access to the accessibility options.

### Additional Context

The closest alternatives are related to the following other tickets:
#3022 - Enable new users to self assign disability badges during the initial setup process
#2892 - Command line utility to execute Resonite bot account commands
#1123 - Help Screen on the Dashboard

### Requesters

Flame Soulis(FlameSoulis), LGK (lunaragk), ChakatSilverpaw (silverpaw), MajorK (vessla)

**Comment by shiftyscales** [comment]
To clarify your request here @FlameSoulis - are you specifically just wanting a mechanism outside of directly inputting Resonite bot account commands to assign accessibility badges? Or compiling all existing accessibility settings under one common tab in the settings UI?

From the description, it sounds like the former, but the title it sounds like the latter.

Either way, we'd need to introduce some mechanism for assigning badges outside of the commands either via #3072 or some other means to handle that request.

As far as just grouping all existing accessibility-related settings together- I'm definitely in favour. The redundancy is useful.

**Comment by FlameSoulis** [comment]
Good evening, Shifty!

More of the latter, specifically **having all existing accessability settings under a common ta**b (even if it has to go under an already existing one like Profile). Having everything under one easy to go to location would greatly improve its usability and could open the doorway on having systems take the accessability options into account (for example, my home world checks for the Potato badge and immediately disables any unneeded items as a 'default to low' indicator).

**Comment by towneh** [comment]
I’m a huge fan of accessibility options being made a lot more obvious to users in the settings UI. It’s vital for Resonite to prioritise removing barriers, and making it a straightforward process, for people being able to use the platform.

**Comment by Frooxius** [comment]
Adding UI for these to be easily settable is definitely something we want to do.

However these aren't really "Settings" so they cannot be added as such. They're user profile options (kinda like bio), which we don't have an UI framework for editing yet.
